#162163256 TravisCI readme badge

joshtrigger opened the pull request:
**What does this PR do?**
Have a TravisCI readme badge

**Description**
This readme badge will display in percentage how much of the code base is e=being covered by the tests.

**What are the relevant Pivotal tracker stories?**
#162163256

Commit:
updated

Commit:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

Commit:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
Merge branch 'ch-coveralls-integration-162163259' into ch-travis-readme-badge-162163256

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
Merge branch 'develop' of https://github.com/andela/ah-backend-athena into develop

joshtrigger committed:
Merge branch 'ch-travis-readme-badge-162163256' of https://github.com/andela/ah-backend-athena into ch-travis-readme-badge-162163256

Kasulejoseph commented:
good work @joshtrigger

collin5 commented on authors/settings.py:
```diff
@@ -78,6 +78,19 @@
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'athenatraviscidb',
+        'USER': 'postgres',
```
Can you please pick these from the `environment`. https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings

joshtrigger replied:
I have worked on the issue

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

Created environment variables on to Travis CI.

[Delivers #162163256]

collin5 reviewed (approved):
LGTM. Thank you!
